cli/config.go

```diff
@@ -144,18 +147,15 @@ func (a *accountSet) MarshalJSON() ([]byte, error) {
 }
 
 func (a *accountSet) UnmarshalJSON(buf []byte) error {
-	var m map[string]struct {
```

Comment:
I'm not sure what I was thinking with these 4 lines, it was very stupid. This would also have broken attempts at parsing configs with old clients too.

This is why you should unit test everything